manuscript/draft.Rmd

```diff
@@ -1,47 +1,58 @@
-# Title
+# Moving towards large-scale data-intensive forecasting of species richness in breeding birds
 
 Authors: Harris, Taylor and White?
 
 ## Introduction
 
-Quality of predictions is important:
-  * helps determine policy/set priorities
-  * something something climate change
-  * need to know both accuracy of point estimates and how reasonable our prediction intervals are.
-  * Overconfident predictions can be worse than uncertainty
-  * Quality of prediction is one metric for how much we know about ecological processes
-Unclear how much to trust existing predictions:
-  * Strong assumptions (see "misc" below)
-  * many *interesting* predictions
-  * Potential problems with confidence intervals
-  * Potential disconnect between training target and actual goals
-  * Difficult to test model assumptions/predictions because of time scales
-    * Most scientists making predictions today will have retired by 2050
-Demonstrate best practices \& their contribution to better predictions
-  * Use long-term data sets such as BBS to evaluate predictive skill
-  * Note the large change in average temperatures since the 80's; comparable to 
-    the potential changed in the next 20 years 
-  * Lots of talk about how we should do it, fewer examples of doing them all together
-  * Examples of best practices
-    * Not just bioclim
-    * Dynamic predictor variables (not just site-level averages over 50 years like 
-      Worldclim)
-    * time series models (temporal autocorrelation)
-    * spatial/site-level stuff (repeated measures aren't fully independent)
-    * Properly handling uncertainty:
-      * probabilistic/error bars
-      * error propagation
-    * multiple models
-      * Test with simple baselines (e.g. random walk, future-is-like-present)
-  * Focus on richness/diversity because \[citations saying that it's important\]
-  * Publish forecasts at different timescales to evaluate performance/credibility
-Here we do that:
-  * forecasts for biodiversity in NAmerican birds with as many best practices as 
-    possible
-  * Discuss the influence of these improved methods \& of our error rates on our 
-    ability to understand:
-    * How to improve
-    * How much to trust the predictions
+Forecasting the future state of ecological systems is increasingly important for
+planning, management, and testing the performance of ecological models
+[@clark2001, @dietz2017]. One area of particular interest for forecasting is
+biodiversity since it is important for ecosystem function, central to
+conservation planning, and expected to be influenced by anthropogenic global
+change. Anticipating potential changes in biodiversity is crucial for large
+scale management and conservation, and for addressing debates regarding whether
+local scale diversity is declining and may continue to decline in response to
+anthropogenic influences [@dornelas2014].
```

Comment:
This might be a good place to mention that quality of prediction is one metric for how much we know about ecological processes. If we can't predict, we [or at least our models] clearly don't understand what's going on. So this isn't just important for applied folks.